### PR TITLE
Update tooling, drop python 2

### PR DESCRIPTION
**Pull request by protoroto**
Update tooling:

- github actions
- pre-commit
- towncrier / bump2version

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -100,16 +90,74 @@ To get tox, just pip install them into your virtualenv.
...
+From now on every commit will be checked against our code style.
+
+Check also the available tox environments with ``tox -l``: the ones not marked with a python version number are tools
+to help you work on the project buy checking / formatting code style, running docs etc.
+
+Testing tips
+------------
+You can test your project using any specific combination of python, django and django cms.
+
+For example ``tox -epy3.7-django30-cms37`` runs the tests on python 3.7, Django 3.0 and django CMS 3.7.
```
Comment (yakky) [inline CONTRIBUTING.rst:112]: adapt environment name to an existing one

File: CONTRIBUTING.rst [hunk 2]
```diff
@@ -100,16 +90,74 @@ To get tox, just pip install them into your virtualenv.
...
+
+#. Pull request must be named with the following naming scheme:
+
+   ``<type>/(<optional-task-type>-)<number>-description``
+
+   See below for available types.
+
+#. The pull request should include tests.
+#. If the pull request adds functionality, the docs should be updated.
+   Documentation must be added in ``docs`` directory, and must include usage
```
Comment (yakky) [inline CONTRIBUTING.rst:127]: edit this as documentation is only included in the readme file

File: CONTRIBUTING.rst [hunk 3]
```diff
@@ -100,16 +90,74 @@ To get tox, just pip install them into your virtualenv.
...
+
+   * ``.feature``: For new features.
+   * ``.bugfix``: For bug fixes.
+   * ``.doc``: For documentation improvement.
+   * ``.removal``: For deprecation or removal of public API.
+   * ``.misc``: For general issues.
+
+   Check `towncrier`_ documentation for more details.
+
+#. The pull request should work for all python / django / django CMS versions
```
Comment (yakky) [inline CONTRIBUTING.rst:143]: remove reference to django / django CMS

File: setup.cfg [hunk 4]
```diff
@@ -1,18 +1,61 @@
...
-combine_as_imports = true
-default_section = THIRDPARTY
-include_trailing_comma = true
-known_first_party = giturlparse
-multi_line_output = 5
-not_skip = __init__.py
\ No newline at end of file
+[options.package_data]
+* = *.txt, *.rst
+djangcms_page_sitemap = *.html *.png *.gif *js *jpg *jpeg *svg *py *mo *po
```
Comment (yakky) [inline setup.cfg:58]: leftover :)

File: setup.cfg [hunk 5]
```diff
@@ -1,18 +1,61 @@
-[flake8]
-exclude = *.egg-info,.git,.settings,.tox,build,dist
-max-line-length = 99
+[bumpversion]
+current_version = 0.8.2.dev1
```
Comment (yakky) [inline setup.cfg:2]: fix current version

**Review by yakky — CHANGES_REQUESTED** [review]
:+1: only minor typos to fix

File: CONTRIBUTING.rst (hunk 1 above)
Comment (protoroto) [reply CONTRIBUTING.rst]: @yakky Fixed

File: setup.cfg (hunk 5 above)
Comment (protoroto) [reply setup.cfg]: @yakky Fixed

File: setup.cfg (hunk 4 above)
Comment (protoroto) [reply setup.cfg]: @yakky Fixed :)

File: CONTRIBUTING.rst (hunk 3 above)
Comment (protoroto) [reply CONTRIBUTING.rst]: @yakky Fixed

**Review by yakky — APPROVED** [review]

**Pull request merged**